mesh/mesh.cpp

```diff
@@ -1041,6 +1041,63 @@ FaceElementTransformations *Mesh::GetBdrFaceTransformations(int BdrElemNo)
    return tr;
 }
 
+void Mesh::GetFaceInformation(int f, FaceInformation &info) const
```

Comment:
This can probably return by value instead of modify the info passed by reference.